src/python/gem5/components/boards/kernel_disk_workload.py

```diff
@@ -145,6 +147,7 @@ def set_kernel_disk_workload(
         disk_image: DiskImageResource,
         bootloader: Optional[BootloaderResource] = None,
         disk_device: Optional[str] = None,
+        additional_disks: List[DiskImageResource] = [],
```

Comment:
Sorry to change my mind here, it's a small suggestion

```suggestion
        additional_disk_images: List[DiskImageResource] = [],
```

It's just a bit clearer and makes the link between it and `disk_image` more apparent.

Also, you should add this to the end of the parameters. If someone has been calling this function with positional arguments then their scripts may break (i.e., it'll look like their trying to pass the `readfile` as `additional_disks`.

Reply:
Done!